### ISSUE TITLE
Update modal to include contributor names

### ISSUE THREAD

**Issue by timhitchins** [antievictionmappingproject/covid-19-map#149]
The opening modal does not currently acknowledge the actual people that were involved in the creation of data, development of the map and translation of text.  The following text should be included: 

```
Map and data by: Azad Amir Ghassemi, Brett Halperin, Chris Henrick, Ciera Dudley, Elana Eden, Erin McElroy, Isnardo Gandarilla, Jim Stewart, Luis Felipe R. Murillo, Paul (Kollectiv Orangotango), Sachi Arakawa, Tim Hitchins, and Wynn Newberry. Translation support: Angie, Daniel Mitchen, İrem Yılmaz, Luis Felipe, Manny, Manon Vergerio, Paul (Kollectiv Orangotango), Jihed Brirmi, Julieta Arancio, Maggie Erving, and Xiaomei.
```

Note: More names will be added in the coming days.  cc @erinmcel

**Comment by timhitchins** [comment]
@erinmcel do you know if the names listed above are complete?  wasn't sure if any got added since last week.  thanks!

**Comment by erinmcel** [comment]
Hi! Thanks for checking. Can we add Samantha Hanae Noh to the first part?

**Comment by timhitchins** [comment]
Addressed on #159
